addons/website_event/models/event_question.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class EventQuestion(models.Model):
+    _name = 'event.question'
+    _rec_name = 'title'
+    _order = 'sequence,id'
+    _description = 'Event Question'
+
+    title = fields.Char(required=True, translate=True, compute='_compute_title', readonly=False, store=True)
+    question_type = fields.Selection([
+        ('simple_choice', 'Selection'),
+        ('text_box', 'Text Input'),
+        ('name', 'Name'),
```

Comment:
Seems those types are new ... should be done in its own commit with explanations, in order to see how the model evolves.